frontend/components/forms/validators/valid_url/valid_url.ts

```diff
@@ -6,8 +6,16 @@ interface IValidUrl {
   url: string;
   /**  Validate protocols specified */
   protocols?: ("http" | "https")[];
+  allowAnyLocalHost?: boolean;
 }
 
-export default ({ url, protocols }: IValidUrl): boolean => {
-  return isURL(url, { protocols });
+export default ({
+  url,
+  protocols,
+  allowAnyLocalHost = false,
+}: IValidUrl): boolean => {
+  if (allowAnyLocalHost && url.includes("localhost")) return true;
```

Comment:
This still allows "localhost.com" etc., which would then be blocked by the back end.  I don't think it's worth the effort to handle that here before we talk to the product team about this though.